package.json

```diff
@@ -25,14 +25,14 @@
   ],
   "license": "Apache-2.0",
   "main": "scripts/build.js",
-  "private": "true",
+  "private": true,
   "repository": {
     "type": "git",
     "url": "git://github.com/koopjs/koopjs.github.io.git"
   },
   "scripts": {
     "build": "sitedown --build . --layout layout.html --watch .",
-    "start": "nodefront serve",
+    "start": "nodefront serve & npm run build",
```

Comment:
yeah, it has the desired effect of spinning up the server and watching the source for changes.  @ungoldman is right that the specific technique i'm proposing is definitely _not_ windows friendly.  

lemme try switching to `npm-run-all`. i don't know if it matters as much here specifically, but i'd be good to get familiar with it and use it to help with on boarding windows esri leaflet devs.

thanks ghostman!